cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/ClusterModelStats.java

```diff
@@ -164,23 +166,24 @@ public String getJSONString() {
   public Map<String, Object> getJsonStructure() {
     Map<String, Object> statMap = new HashMap<>();
     Map<String, Integer> basicMap = new HashMap<>();
-    basicMap.put("brokers", numBrokers());
-    basicMap.put("replicas", numReplicasInCluster());
-    basicMap.put("topics", numTopics());
+    basicMap.put(AnalyzerUtils.BROKERS, numBrokers());
+    basicMap.put(AnalyzerUtils.REPLICAS, numReplicasInCluster());
+    basicMap.put(AnalyzerUtils.TOPICS, numTopics());
+    List<Statistic> cachedStatistic = Statistic.cachedValues();
     // List of all statistics AVG, MAX, MIN, STD
-    Map<String, Object> allStatMap = new HashMap();
-    for (Statistic stat : Statistic.values()) {
+    Map<String, Object> allStatMap = new HashMap<>(cachedStatistic.size());
+    for (Statistic stat : cachedStatistic) {
       Map<String, Object> resourceMap = new HashMap<>();
```

Comment:
we can also initialize the capacity to `cachedStatistic.size()+3`.

Reply:
I assume you mean `cachedResources` -- i.e. not `cachedStatistic`.